docs/knowledge_base/tips/dataiku/index.md

```diff
@@ -0,0 +1,255 @@
+apiKey = "your_api_key"
+
+client = DSSClient(host, apiKey)
+```
+
+## Retrieving Data from Dataiku
+
+Fetch data from your Dataiku projects by iterating through the projects and collecting relevant metrics for each model. This example code retrieves metrics like AUC and organizes them into a `pandas.DataFrame`.
+
+```python
+metrics_table = []
+for project_keys in client.list_project_keys():
+    project = client.get_project(project_keys)
+    project_label = project.get_metadata()["label"]
+    for model_id in project.list_saved_models():
+        model = project.get_saved_model(model_id["id"])
+        try:
+            active_id = model.get_active_version().get("id", None)
+            metrics = model.get_metric_values(active_id)
+            for metric in metrics.raw['metrics']:
+                if metric['metric']['metricType'] == "AUC":
+                    value = metric['lastValues'][0]['value']
+                    metrics_table.append({'Project': project_label,
+                                          'ModelId': model_id['id'],
+                                          'ActiveId': active_id,
+                                          'MetricType': "AUC",
+                                          'Value': value})
+        except Exception as e:
+            print(e)
+            pass
+
+metrics_df = pd.DataFrame(metrics_table)
+```
+
+## Preparing Data for Visualization
+
+Before visualization, you may need to further prepare your data. For instance, you can concatenate certain columns to form a unique identifier or name for each entry. Here, we add a 'Name' column to our DataFrame.
+
+```python
+metrics_df['Name'] = metrics_df[['Project', 'ModelId', 'MetricType']].agg(' - '.join, axis=1)
+```
+
+## Creating and Running a Taipy Application
+
+Define the GUI using Taipy's markdown-like syntax. This GUI will include dynamic selectors for projects and metrics, a table to display data, and a bar chart for visualizing metric values.
+
+```python
+from taipy.gui import Gui
+
+selected_projects = list(metrics_df['Project'].unique())
+selected_metrics = ["AUC"]  # Initially focusing on AUC metrics.
+
+def show_projects(metrics_df, selected_projects, selected_metrics):
+    return metrics_df[(metrics_df['Project'].isin(selected_projects)) &
+                      (metrics_df['MetricType'].isin(selected_metrics))]
+
+md = """
+# Dataiku **Overview**{: .color-primary}
+
+**Projects**
+
+<|{selected_projects}|selector|lov={list(metrics_df['Project'].unique())}|dropdown|multiple|class_name=fullwidth|label=Projects|>
+
+**Metrics**
+
+<|{selected_metrics}|selector|lov={list(metrics_df['MetricType'].unique())}|dropdown|multiple|class_name=fullwidth|label=Metrics|>
+
+<|{show_projects(metrics_df, selected_projects, selected_metrics)}|table|filter|>
+
+<|{show_projects(metrics_df, selected_projects, selected_metrics)}|chart|x=Name|y=Value|type=bar|>
+"""
+
+Gui(md).run()
+```
+
+
+
+# Dataiku Projects with Taipy Scenarios
```

Comment:
Can you have a title showing the motivation ?

"Run a Dataiku pipeline from your Taipy application"
"Let your end-user run a Dataiku pipeline"
etc.